api-gateway/src/main/resources/static/scripts/bill-details/bill-details.template.html

```diff
@@ -0,0 +1,39 @@
+<script>
+    function printPage() {
+        window.print();
+    }
+</script>
+
+<!--<h2>Bill Details: {{bills.id}}</h2>
+
+<div>
+    <h3>Payment Info for {{bills.billId}}</h3>
+    <h3>505 Lincoln Str. Queens, New York</h3>
+    <h3>{{owners.firstName}} {{owners.lastName}}</h3>
+    <h3>Final price: {{bills.amount}}</h3>
+    <h3>Visit type: {{bills.visitType}}</h3>
+    <h3>Date generated: {{bills.date}} {{bills.time}}</h3>
+
+    <div>
+        <input type="button" value="Print this page" onclick="printPage()" />
+    </div>
+
+    <button type="submit" formaction="/bills" >Back to History...</button>
+</div>-->
```

Comment:
Should we be pushing this commented code to main? Maybe we could implement this in a later story.